Tools/install.sh

```diff
@@ -22,6 +22,46 @@ FONT_PACKAGES=(\
     fonts-droid-fallback \
     )
 
+check_for_installed_tools () {
+    REQUIRED_TOOLS="${@}"
+    echo -e "Checking for installed tools.\nrequirded tools: ${REQUIRED_TOOLS}"
```

Comment:
`requirded` -> `Required`

Reply:
This is just a style thing but does `${REQUIRED_TOOLS}` have any advantages over `$REQUIRED_TOOLS`? I'm looking at consistency here.

Reply:
Curly braces are not making a difference for the above use case, but are required in other cases:

- accessing arrays
- using substitution functions
- combining variable content with fixed value ("${foo}bar").

I am used to using them everywhere for consistency :-). But yes here it's general use is inconsistent.